source/js/buyers-guide/components/primary-nav/primary-nav.scss

```diff
@@ -1,4 +1,5 @@
 $pni-primary-nav-breakpoint: $bp-sm;
+$pni-primary-nav-container-z-index: 30;
```

Comment:
is `30` based on something, or should we try to relate this to the bootstrap z-index values (https://getbootstrap.com/docs/4.0/layout/overview/#z-index) ?

Reply:
`30` was from existing code. But I can update it to use Bootstrap variable.

Reply:
Updated!